docs/codeql/codeql-for-visual-studio-code/analyzing-your-projects.rst

```diff
@@ -129,12 +129,14 @@ To remove queries from the Query History view, select all the queries you want t
 Viewing query results
 -----------------------
 
-#. Click a query in the Query History view to display its results in the Results view. Alternatively, right-click the result to select a different view:
+#. Click a query in the Query History view to display its results in the Results view.
 
-   - To view the results in :ref:`SARIF format <sarif-output>`, right-click and select **View SARIF**.
-   - To view the results in :ref:`DIL format <dil>`, right-click and select **View DIL**.
+   .. pull-quote:: Note
 
-#. Use the dropdown menu in the Results view to choose which results to display, and in what form to display them, such as a formatted alert message or a table of raw results. The available output forms are specified by the format of the query and the metadata. For more information, see ":ref:`CodeQL queries <codeql-queries>`."
+      Depending on the query, you can also choose different views such as CSV, :ref:`SARIF <sarif-output>`, or :ref:`DIL format <dil>`. For example, to view the DIL format, right-click a result and select **View DIL**.
```

Comment:
Is there a logic in the order in which the formats are listed (Is it how they appear when you right-click a result?) or should we mention them alphabetically?

Reply:
Good question! I've roughly listed them in order of "usefulness" 😀 People are probably most likely to want CSV, then SARIF, and (rarely) DIL. I think this also corresponds to the UI, but it depends on what's available, given the format of the query.